package.json

```diff
@@ -25,41 +25,34 @@
     "clean": "node_modules/.bin/rimraf dist internal types *.{d.ts,js}{,.map} build-test tsconfig.tsbuildinfo"
   },
   "files": [
-    "dist/src/",
-    "src/",
```

Comment:
I think the`src` folder should be listed here. Installing with `npm` from the branch will result in a the `node_modules/@hashgraph/hardhat-hethers` folder containing only package.json, license and readme files. Also the dist folder does not get generated with both `npm` and `yarn` (although with yarn the `src` folder is present)

Reply:
I added a `prepublishOnly` script to auto run the build command before publish. This will make sure that `npm run build` is always run when publishing a new version. And I added the `src` directory here which will allow installing the plugin directly from the repo, but I think you will still need to manually navigate to the plugin dir in node_modules and run the build command.

Reply:
Yeah, prepublishOnly will run build when publishing the package, I think `prepublish` or `prepare` will run on both install and publish. Tbf I'm not sure which one is most appropriate